Jenkinsfile

```diff
@@ -61,18 +62,19 @@ node('vetsgov-general-purpose') {
           if (!shouldBuild) { return }
 
           try {
+            // try to build using fresh drupal content
```

Comment:
The entire contents of this outer `try...catch` is copy-pasted for each `envName`. [I tried to extract it, but ran into an error](https://github.com/department-of-veterans-affairs/vets-website/pull/16965#discussion_r621687442), and wanted to limit the scope/risk of the change. 

>prefer duplication over the wrong abstraction - https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction